src/translations.js

```diff
@@ -33,6 +34,9 @@ const statCardLocales = ({ name, apostrophe }) => {
       ar: "مجموع النجوم",
       cn: "获标星数（star）",
       cs: "Celkem hvězd",
+      de: "Sterne Insgesamt",
```

Comment:
@FayasNoushad Thanks a lot for your contribution. Please remove the `en` and `de` changes as we currently only allow one language per pr.